backend/src/main/resources/application.properties

```diff
@@ -2,5 +2,3 @@ spring.application.name=backend
 spring.docker.compose.file=backend/compose.yaml
 
 server.port=9000
-spring.security.oauth2.client.registration.google.client-id=
-spring.security.oauth2.client.registration.google.client-secret=
```

Comment:
Should we add something like this and have env. variables in our editor? 
spring.security.oauth2.client.registration.google.client-id=${GOOGLE_CLIENT_ID}
spring.security.oauth2.client.registration.google.client-secret=${GOOGLE_CLIENT_SECRET}

Reply:
Yes, that would be nice! I have added 
spring.security.oauth2.client.registration.google.client-id=${ID}
spring.security.oauth2.client.registration.google.client-secret=${SECRET}
to my project now, but I haven't pushed it. Shall I push this change as well?